infrastructure/templates/infrastructure/_project_transmission_list.html

```diff
@@ -0,0 +1,42 @@
+<table>{% load humanize %}
```

Comment:
I think we should make a couple small CSS adjustments to get the table looking better on mobile devices (it doesn't look broken under realistic conditions, but it also has some rough edges):

Somewhere in there is a rule for `article section table thead th:first-child` elements that gives them `padding-right: 6px`. When the name of the location is big, this ends up looking cramped:

![image](https://user-images.githubusercontent.com/1625749/53446632-6bc4df80-39e1-11e9-836d-12e379a48062.png)

If we change it to `:not(:last-child)` (or some other less awkward version of that), it's less so:

![image](https://user-images.githubusercontent.com/1625749/53446662-8303cd00-39e1-11e9-82a9-469e4b5b33ca.png)

Also, in general, it does get kinda cramped-looking when the location name is big, and when it's freakishly big (arguably unrealistically, but we're probably dealing with some pretty dirty data), the layout breaks:

![image](https://user-images.githubusercontent.com/1625749/53446741-a62e7c80-39e1-11e9-854d-4d6ce83c2e76.png)

I don't really know what we can do about that. We wouldn't want to generalize to the worst case by introducing word-breaking or forbidding wrapping & introducing ellipses. But it also seems bad to knowingly let the layout break.

It'd be nice to use something like Bootstrap's "responsive tables", which are essentially just wrapper elements that have `overflow-x: auto` to allow the user of a mobile device to scroll side to side in the table as needed. That might be a good thing to introduce here.